src/library_pthread.js

```diff
@@ -4,6 +4,28 @@
  * SPDX-License-Identifier: MIT
  */
 
+#if !USE_PTHREADS
+#error Internal error! USE_PTHREADS should be enabled when including library_pthread.js.
+#endif
+#if !SHARED_MEMORY
+#error Internal error! SHARED_MEMORY should be enabled when including library_pthread.js.
+#endif
+#if USE_PTHREADS == 2
+#error USE_PTHREADS=2 is no longer supported
+#endif
+#if BUILD_AS_WORKER
+#error USE_PTHREADS + BUILD_AS_WORKER require separate modes that don\'t work together, see https://github.com/emscripten-core/emscripten/issues/8854
+#endif
+#if PROXY_TO_WORKER
+#error --proxy-to-worker is not supported with -s USE_PTHREADS>0! Use the option -s PROXY_TO_PTHREAD=1 if you want to run the main thread of a multithreaded application in a web worker.
+#endif
+#if EVAL_CTORS
+#error EVAL_CTORS is not compatible with pthreads yet (passive segments)
+#endif
+#if STANDALONE_WASM
+#error STANDALONE_WASM does not support shared memories yet
```

Comment:
Do you think quoting the error strings makes them more readable?  e.g. `#error "STANDALONE_WASM does not support shared memories yet"`.    I think at least it will make the syntax highlighting work.

Reply:
Added quotes